packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -622,7 +622,11 @@ class RenderParagraph extends RenderBox with ContainerRenderObjectMixin<RenderBo
     return getOffsetForCaret(position, Rect.zero) + Offset(0, getFullHeightForCaret(position) ?? 0.0);
   }
 
-  List<ui.LineMetrics> _computeLineMetrics() {
+  /// Returns the full list of [LineMetrics] that describe in detail the various
+  /// metrics of each laid out line.
```

Comment:
Instead of adding these docs, could you make the docs for TextPainter.computeLineMetrics a `macro` and then use it here?